plugins/push/frontend/public/templates/push-notification-app-level-config.html

```diff
@@ -59,4 +70,14 @@
     <cly-inline-form-field label="Period duration (seconds)" rules="numeric">
         <el-input :value="viewModel.period" @input="onInput('period',$event)"></el-input>
     </cly-inline-form-field>
+    <cly-confirm-dialog 
+        title="Delete key"
+        dialogType="danger"
+        saveButtonLabel="I understand, delete this key" 
+        cancelButtonLabel="Cancel" 
+        :visible.sync="isDeleteKeyDialogVisible"
+        @cancel="onDeleteKeyCancel" 
+        @confirm="onDeleteKeyConfirm">
+    </cly-confirm-dialog>
+</cly-confirm-dialog >
```

Comment:
You can use CountlyHelpers.confirm unless you have a specific reason not to do so :slightly_smiling_face:

Reply:
yes I will. It slipped my mind.